app/views/index.scala.html

```diff
@@ -269,6 +286,60 @@
 <link href='@routes.Assets.at("stylesheets/choropleth.css")' rel="stylesheet"/>
 <script type="text/javascript" src='@routes.Assets.at("javascripts/Choropleths/Choropleth.js")'></script>
 
+@*    overlapping navbar    *@
```

Comment:
Let's delete the style that we don't want to include (it's now in the commit history so we can find it in the future easily enough). And the styling should be moved to homepage.css; according the [style guide](https://github.com/ProjectSidewalk/SidewalkWebpage/wiki/Style-guide#htmlcss-guidelines), CSS should be in dedicated CSS files instead of the HTML!